frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -125,14 +125,15 @@ const handleNameChange = (enteredName, fieldErrors, setName, setFieldErrors) =>
   const nameValidation = validateDNS1123SubdomainValue(enteredName, {
     // t('kubevirt-plugin~Network attachement definition name cannot be empty')
```

Comment:
I just realize that "attachment" is spelled wrong multiple times in this file. It's currently spelled "attachEment", but that 'e' shouldn't be there.